testsuite/documentation/cucumber-steps.md

```diff
@@ -322,7 +322,7 @@ The check box can be identified by name, id or label text.
   When I check the row with the "sle-client" hostname
   When I check the row with the "suse_docker_admin" text
   When I check "New Test Channel" in the list
-  When I uncheck "hoag-dummy-1.1-2.1" in the list
```

Comment:
Silly question, why we downgraded the package version, instead of use a higher number?

Reply:
Because we changed to a new repository and therefore it restarted at version 1.1.

As a matter of fact, this 2.1 value was an "accident" we did not know how to fix.